ViewGenerator/ViewGenerator.nuspec

```diff
@@ -2,14 +2,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>ViewGenerator</id>
-    <version>1.0.290</version>
+    <version>1.0.293</version>
```

Comment:
why 3 versions ahead?

Reply:
I just incremented every time I needed to raise a version during development. Is that a problem?

Reply:
there's no harm to it
but you just need it once to update it in SS

Reply:
It's ok, we usually do that as well during development